### PR TITLE
Add stryker configuration.

### PR DESCRIPTION
**Pull request by ruiaraujo**
Stryker allows us to have a sense of the quality of our tests.

It does take a while to run so no automation for now.

### CODE REVIEW

**Comment by ruiaraujo** [comment]
👍

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "lint-fix": "tslint --project . --fix",
     "build": "tsc",
     "test": "jest",
+    "stryker": "stryker run",
```
Comment (boopathi) [inline package.json]: nit: this takes the striker namespace in commands and we can't do `yarn stryker --somethingelse`. Change it to a different name.

File: stryker.conf.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module.exports = function(config) {
+  config.set({
+    mutator: "typescript",
+    packageManager: "yarn",
+    reporters: ["clear-text", "progress"],
+    testRunner: "jest",
+    transpilers: [],
+    coverageAnalysis: "off",
+    tsconfigFile: "tsconfig.json",
+    mutate: ["src/**/*.ts",
```
Comment (boopathi) [inline stryker.conf.js]: nit: this is not formatted with prettier

**Comment by vigneshshanmugam** [comment]
@ruiaraujo https://www.stryker.de/ You got me thinking for a bit 😌

**Comment by addityasingh** [comment]
👍

**Comment by ruiaraujo** [comment]
:+1:

**Comment by ruiaraujo** [comment]
@vigneshshanmugam This mutator is pretty good to show the quality of the tests.
graphql-jit scores 85% which is okay.

**Comment by boopathi** [comment]
👍

**Pull request merged**